curiouslearning/FeedTheMonsterJS
# fix: Cr 170 Oromo language  rebuild

# Changes
- ftm_oromo.json

# How to test
- Check oromo language levels and words.

Ref: [CR-170](https://curiouslearning.atlassian.net/jira/software/projects/CR/boards/7?selectedIssue=CR-170)


[CR-170]: https://curiouslearning.atlassian.net/browse/CR-170?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
